src/main/java/net/sf/mzmine/modules/visualization/ida/IDADataSet.java

```diff
@@ -274,12 +275,11 @@ public void setColor(int series, int item, Color c) {
      *            color.
      * 
      */
-    public void highlightSpectra(double mz, double ppm, int minIntensity,
```

Comment:
By the way, we normally use double type for intensity, I see minIntensity is int here. Is there any reason for that? Remember that int has a fixed range of allowed values -2,147,483,648 to 2,147,483,647, so if you get to intensities above 2E9 you might get overflows (2E9 values are commonly seen e.g. on Q Exactive).